Resources/Prototypes/_NF/Catalog/Fills/Items/weapon_cases_expedition.yml

```diff
@@ -107,6 +107,22 @@
     - id: SpeedLoaderMagnum
       amount: 2
 
+- type: entity
+  parent: WeaponCaseShort
+  id: WeaponCaseShortTheArbitrator
+  noSpawn: true
```

Comment:
Needs to be replaced with  `categories: [ HideSpawnMenu ]`

Reply:
done